sql/src/main/java/org/apache/druid/sql/avatica/ErrorHandler.java

```diff
@@ -47,58 +49,67 @@
   }
 
   /**
-   * Logs a throwable at the error level and sanitizes the throwable if applicable. Will return
-   * the sanitized or original throwable.
+   * Logs any throwable and string format message with args at the error level.
    *
-   * @param error A throwable that will be logged then sanitized
-   * @param <T>
-   * @return The sanitized throwable
+   * @param error   the Throwable to be logged
+   * @param message the message to be logged. Can be in string format structure
+   * @param format  the format arguments for the format message string
+   * @param <T>     any type that extends throwable
+   * @return the original Throwable
    */
-  public <T extends Throwable> RuntimeException logFailureAndSanitize(T error)
+  public static <T extends Throwable> T logFailure(T error, String message, Object... format)
```

Comment:
why do we need logFailure methods instead of just having the caller directly calling LOG.error?

Reply:
This was how @clintropolis did it. i reverted my changes to them and just moved them back to `DruidMeta` as they aren't necessary here anymore